human_analysis/logs/NN_2500_30/tybalt_2layer_30latent_stats.tsv

```diff
@@ -1,41 +1,41 @@
 val_loss	loss	learning_rate	batch_size	epochs	kappa
-3470.9262459047377	3939.4017039512682	0.001	10	40	0.01
-3332.357177734375	3490.503589059424	0.001	10	40	0.01
-3301.504945816532	3432.473061170421	0.001	10	40	0.01
-3286.100050403226	3396.6803260754073	0.001	10	40	0.01
-3243.4317981350805	3368.998511388802	0.001	10	40	0.01
-3226.9675214213707	3325.6142789585383	0.001	10	40	0.01
-3212.0390625	3299.4588110124373	0.001	10	40	0.01
-3192.759230090726	3282.9244102527177	0.001	10	40	0.01
-3203.371077998992	3261.168110655646	0.001	10	40	0.01
-3177.0357862903224	3247.7635069009484	0.001	10	40	0.01
-3170.5570580267135	3217.010652360527	0.001	10	40	0.01
-3158.702589465726	3212.6116215481275	0.001	10	40	0.01
-3165.5476783014115	3195.0789330932357	0.001	10	40	0.01
-3135.946304813508	3190.7616121460046	0.001	10	40	0.01
-3129.0841261340724	3175.6680007311984	0.001	10	40	0.01
-3113.454558341734	3157.7240807240405	0.001	10	40	0.01
-3105.3703849546373	3155.2653284311805	0.001	10	40	0.01
-3099.3141853578627	3143.7844752077553	0.001	10	40	0.01
-3094.0861737651207	3137.6941898375985	0.001	10	40	0.01
-3099.5052765877017	3135.4627474960294	0.001	10	40	0.01
-3091.8208952872983	3125.0009578631107	0.001	10	40	0.01
-3087.291322769657	3109.552629693372	0.001	10	40	0.01
-3092.7088583669356	3115.570717944703	0.001	10	40	0.01
-3077.6540606098793	3093.6086710641107	0.001	10	40	0.01
-3074.8664157006046	3092.215459606523	0.001	10	40	0.01
-3067.576234879032	3099.684976805767	0.001	10	40	0.01
-3072.006623298891	3093.5772038366254	0.001	10	40	0.01
-3058.075864730343	3077.059732490381	0.001	10	40	0.01
-3049.378843245968	3078.525634765625	0.001	10	40	0.01
-3061.21484375	3065.8411287650433	0.001	10	40	0.01
-3050.4500299269152	3067.0633026756555	0.001	10	40	0.01
-3054.2125204763106	3062.1776258486375	0.001	10	40	0.01
-3051.246826171875	3063.690449085932	0.001	10	40	0.01
-3044.867250504032	3053.3639434180945	0.001	10	40	0.01
-3036.5818737399195	3048.1008814577553	0.001	10	40	0.01
-3046.1041535408267	3044.620108274707	0.001	10	40	0.01
-3044.0268160912296	3040.546544527615	0.001	10	40	0.01
-3033.2976310483873	3044.9473135094177	0.001	10	40	0.01
-3034.587843371976	3032.0526735582944	0.001	10	40	0.01
-3036.5941043976813	3029.345461256263	0.001	10	40	0.01
+2816.8391862709377	3013.397590357973	0.001	10	40	0.01
```

Comment:
I assume this project was ran using a different dataset. The loss seems to be quite better than the previous run. Is this correct?

Reply:
The dataset I used originally, that was part of the grant, was a smaller dataset of ~1.5K samples and 10% were held out for validation. This new dataset is ~5K samples with 25% held out for validation. So the new loss is lower, though there is still a fair gap between the validation and training loss that hopefully can be reduced with more tuning in the future